sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/LifeTimeAction.java

```diff
@@ -35,10 +35,10 @@
 
     /**
      * Creates a new LifetimeAction instance, with the provided {@link CertificatePolicyAction}.
-     * @param certificatePolicyAction The action type of this LifetimeAction.
+     * @param action The action type of this LifetimeAction.
      */
-    public LifeTimeAction(CertificatePolicyAction certificatePolicyAction) {
-        this.certificatePolicyAction = certificatePolicyAction;
+    public LifeTimeAction(CertificatePolicyAction action) {
```

Comment:
The name of the class should be "LifetimeAction" ("lifetime" is one word and the "T" should not be upper-cased).

Reply:
yup, changed it.